scripts/release.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+if [ -z "$OLD_VERSION" ]; then
+    echo "A release requires OLD_VERSION to be defined"
+    exit 1
+fi
+
+if [ "$OLD_VERSION" != "none" ]; then
+    comparison="$OLD_VERSION..HEAD"
+fi
+pre_release="-p"
+
+if [ "$OLD_VERSION" != "none" ];  then
+    changelog=$(git log $comparison --oneline --no-merges --reverse)
+
+    if [ -z "$changelog" ]; then
+        echo "No new changes to release!"
+        exit 0
+    fi
+else
+    changelog="don't forget to update the changelog"
+fi
+
+set -x
+( ( github-release -v release $pre_release -r auth_proxy -t $BUILD_VERSION -d "**Changelog**<br/>$changelog" ) ) || exit 1
```

Comment:
Do you see a situation where a non-dev is doing a release?  We could also pull one of their [formal releases](https://github.com/aktau/github-release/releases), but then it has to be OS-aware...

Maybe we can just do:

```sh
if [ -n "which github-release`" ]; then
  echo >&2 "Please install github-release before running this script"
  echo >&2 "You may download a release from https://github.com/aktau/github-release/releases or run `go get github.com/aktau/github-release` if you have Go installed"
  exit 1
fi
```

Rest looks fine to me.